d3/d3.d.ts

```diff
@@ -549,10 +585,15 @@ declare module D3 {
         functor<R,T>(value: (p : R) => T): (p : R) => T;
         functor<T>(value: T): (p : any) => T;
 
-        map(): Map<any>;
-        set(): Set<any>;
-        map<T>(object: {[key: string]: T; }): Map<T>;
-        set<T>(array: T[]): Set<T>;
+        map:{
+        	<T>():Map<T>;
+        	<T>(object: {[key: string]: T; }) : Map<T>;
+        	<T>(object: Array<T>, accessor?: (d: T, index?: number) => T) : Map<T>;
+        }
+        set:{
+        	<T>():Set<T>;
+        	<T>(array: T[]):Set<T>;
+        }
```

Comment:
Not sure about DefinitelyTyped cares about style, but a consistent spacing heuristic here would make this part easy to read.